v1.1/certificate-rotation.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Rotate Security Certificates
+summary: Rotate the security certificates of a secure CockroachDB cluster by generating and reloading new certificates.
+toc: false
+---
+
+<span class="version-tag">New in v1.1:</span> CockroachDB allows you to rotate security certificates without restarting the nodes. 
+
+<div id="toc"></div>
+
+## How CockroachDB Security Certificates Work
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys:
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Server certificate
+`node.key`   | Key for server certificate
+`client.<user>.crt` | Client certificate for `<user>` (eg: `client.root.crt` for user `root`)
+`client.<user>.key` | Key for the client certificate
+
+The node and client certificates are signed by the CA certificate. 
+
+For each node, the CA certificate, node certificate, and node keys are uploaded to certs directory on the node. For each client, the CA certificate, client certificate, and client keys are uploaded to the client. 
```

Comment:
I think mentioning the CA key is necessary. Probably best to keep it outside the `name | usage` table and emphasize that it is *NOT* copied to the nodes or clients.